src/core/Base32.cpp

```diff
@@ -195,14 +199,15 @@ QByteArray Base32::encode(const QByteArray& data)
             n = 20;
             break;
         default: // expand to 35 bits
-            Q_ASSERT(rBits == 32);
+            Q_ASSERT(32 == rBits);
```

Comment:
Any particular reason for Yoda conditions? They look out of place since the rest of the project doesn't use it.

Reply:
I use them sometimes.

Reply:
@weslly No other reason than personal preference. 

![yoda-emoji](http://www.yodaspeak.co.uk/yoda-small1.gif) Keep it in mind, I will.  My intention to differ from the project's codingstyle it is not. 😄